include/algorithms/boosting/boosting_predict.h

```diff
@@ -74,10 +74,10 @@ namespace interface1
  *      - \ref classifier::prediction::interface1::Input "classifier::prediction::Input" class
  *      - \ref classifier::prediction::interface1::Result "classifier::prediction::Result" class
  */
-class Batch : public classifier::prediction::Batch
+class Batch : public classifier::prediction::interface1::Batch
```

Comment:
Does it mean that this class is deprecated as there is no interface2::Batch?

Reply:
boosting namespace is fully deprecated